apps/teams-test-app/src/components/privateApis/ExternalAppAuthenticationForCEAAPIs.tsx

```diff
@@ -161,6 +161,8 @@ const AuthenticateWithSSOAndResendRequestForCEA = (): React.ReactElement =>
     conversationId: string;
     authTokenRequest: externalAppAuthentication.AuthTokenRequestParameters;
     originalRequestInfo: externalAppAuthentication.IActionExecuteInvokeRequest;
+    authId: string;
```

Comment:
Can authId and connectionName be part of an interface that extends AuthTokenRequestParameters?

Reply:
done!